### ISSUE TITLE
NHL, Class = Teams, properties = goals_against and total_goals_against both return "None"

### ISSUE THREAD

**Issue by conner-mcnicholas** [roclark/sportsipy#665]
**Describe the bug**
Within the NHL module, the Teams class has at least a couple properties that return "None" instead of the correct int.

**To Reproduce**
```python
from sportsipy.nhl.teams import Teams

for year in range(2018, 2021):
    teams = list(Teams(year))
    statd = {}
    for team in teams[0:len(teams)]:
        print('%s F:%s A:%s Avg:%s' % (team.abbreviation,team.goals_for,team.goals_against,team.total_goals_per_game))
```

**Expected behavior**
Output from the snippet above are shown below.  The expected behavior is demonstrated in the "F" int values, as the "goals_for" property is working as expected, unlike the "A" and "Avg" values.

> NSH F:261 A:None Avg:None
> TBL F:290 A:None Avg:None
> ...(truncated)

**Desktop:**
 - OS: Chromebook 
 - Jupyter Notebook running in chrome browser launched from Linux:
 - Linux penguin 5.4.131-16049-gc688e114d34e #1 SMP PREEMPT Thu Sep 9 20:21:35 PDT 2021 x86_64 GNU/Linux
 - Sportsipy Version: 1

**Additional context**
This occurs across all seasons and all teams.  I checked the hockey-reference website and the nominal values appear normal there.

**Comment by david-hosk** [comment]
I'm also experiencing this issue. I submitted another issue before seeing this thread. I am very new to Python and web scraping, but could this issue have something to do with the HTML formatting of the sports-reference website? In other words, has the source code of the website changed in a way to break the sportsipy code?

**Comment by conner-mcnicholas** [comment]
> Could this issue have something to do with the HTML formatting of the sports-reference website? In other words, has the source code of the website changed in a way to break the sportsipy code?

Nope, that's not what is happening.  My workaround was to just use the urllib.request and BeautifulSoup modules to retrieve and parse the sports-reference html directly, just as a one-off for these buggy fields.  So that rules out any issue with the data source.

Here is some of that workaround code if you want to use it:

https://github.com/conner-mcnicholas/jupyter_notebooks/blob/30033f0b2a2fceb892000b6a24f263a518c1916e/SportsAnalysis/ScoreRatio%20vs%20WinRatio%20Correlation%20-%20NHL.ipynb
